beaconclient/multi_beacon_client.go

```diff
@@ -255,10 +259,20 @@ type publishResp struct {
 }
 
 // PublishBlock publishes the signed beacon block via https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/publishBlock
-func (c *MultiBeaconClient) PublishBlock(block *common.SignedBeaconBlock) (code int, err error) {
+func (c *MultiBeaconClient) PublishBlock(block *common.VersionedSignedProposal) (code int, err error) {
+	slot, err := block.Slot()
+	if err != nil {
+		c.log.WithError(err).Warn("failed to publish block as block slot is missing")
+		return 0, err
+	}
+	blockHash, err := block.ExecutionBlockHash()
+	if err != nil {
+		c.log.WithError(err).Warn("failed to publish block as block hash is missing")
+		return 0, err
+	}
 	log := c.log.WithFields(logrus.Fields{
-		"slot":      block.Slot(),
-		"blockHash": block.BlockHash(),
+		"slot":      slot,
+		"blockHash": blockHash.String(),
```

Comment:
why would it coerce into string? wouldn't it log [12, 23, 30 ...]?

Reply:
logrus.Fields is `type Fields map[string]interface{}` - I'm not sure if its formatted as a string here

Reply:
Ah dang you're totally right. Sorry about that. It would print like this without the `.String()`:

```
blockHash="[1 2 3 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]"
```